sql/updates/world/2015_07_02_00_world.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE `creature_template` SET `ScriptName`='npc_flesh_eating_insect' WHERE  `entry`=37782;
+UPDATE `creature_template` SET `ScriptName`='npc_putricide_trap' WHERE  `entry`=38879;
+DELETE FROM `spell_script_names` where `spell_id` IN (70484,70485,70475);
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES 
+(70484, 'spell_icc_summon_plagued_insect'),
+(70475, 'spell_icc_giant_swarm');
```

Comment:
It looks like you are missing conditions for 70485 targeting
Targets (46, 86) (TARGET_DEST_NEARBY_ENTRY, TARGET_DEST_DEST_RANDOM)

It probably should select one of the two triggers in the middle (and then DEST_RANDOM targeting offsets that position within 10 yards, randomly)